app/components/layout.tsx

```diff
@@ -14,7 +14,7 @@ import { Link, useLocation } from "@remix-run/react";
 import Sidebar from "~/components/sidebar";
 import { Dialog, Transition } from "@headlessui/react";
 import Search from "~/components/search";
-import useKeyboardShortcut from "use-keyboard-shortcut";
+// import useKeyboardShortcut from "use-keyboard-shortcut";
```

Comment:
For whatever reason it has trouble importing this, guessing due to the confusion between server and client.